src/tir/transforms/lower_tvm_builtin.cc

```diff
@@ -73,10 +84,19 @@ class StackSizeChecker : public StmtExprVisitor {
       return MakeShape(op);
     } else if (op->op.same_as(builtin::tvm_stack_make_array())) {
       return MakeArray(op);
+    } else if (op->op.same_as(builtin::mem_copy())) {
+      return MakeMemCopy(op);
```

Comment:
Can't you just call `MakeCallPacked` for this?

Reply:
Not quite, because the additional string for the name of the PackedFunc to execute takes up another spot on the stack.  But since I needed to test it in order to convince myself of that, I've added a comment to explain the reasoning.

Reply:
I also figured that by having the `MakeMemCopyHelper` be shared between `StackSizeChecker` and `BuiltInLower` makes it harder for them to get out of sync in the future.

Reply:
That makes sense.  Does that mean that the name "nonexistent_function" is actually significant in the sense that its length equals or exceeds the length of the actually called function name?  That would be worth a comment (if that's the case).

Reply:
Good question to ask, and no, the function name isn't significant.  The stack size here is in number of arguments, where a string only counts as a single argument.